django/contrib/gis/geos/geometry.py

```diff
@@ -292,6 +292,14 @@ def contains(self, other):
         "Returns true if other.within(this) returns true."
         return capi.geos_contains(self.ptr, other.ptr)
 
+    def covers(self, other):
+        """
+        Returns true if the DE-9IM Intersection Matrix for the two geometries
```

Comment:
Do you think it worth to make docstrings consistent in the whole file? If so, I could create PR for this.